conans/cli/commands/lock.py

```diff
@@ -61,3 +65,27 @@ def lock_merge(conan_api, parser, subparser, *args):
     lockfile_out = make_abs_path(args.lockfile_out)
     result.save(lockfile_out)
     ConanOutput().info("Generated lockfile: %s" % lockfile_out)
+
+
+@conan_subcommand()
+def lock_add(conan_api, parser, subparser, *args):
+    """
+    Merge 2 or more lockfiles
+    """
+    subparser.add_argument('--requires', action="append", help='Path to lockfile to be merged')
```

Comment:
Fix the help. What is this argument requiring exactly? has to be "name/version" or could it be any valid reference? with revision too? Would it make sense a package reference too?